src/components/Calculator.js

```diff
@@ -1,38 +1,51 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+// import operate from '../logic/operate';
```

Comment:
- [ ] Kindly remove commented code from your code to make it clean

Reply:
Thank you, sir, for raising my attention to this.